themes/theme-gmd/pages/Product/components/Options/index.jsx

```diff
@@ -74,20 +75,33 @@ class Options extends PureComponent {
           {(options !== null) && (
             <div data-test-id="optionsPicker">
               {options.map((option) => {
-                if (option.type !== 'select') {
-                  return null;
+                if (option.type === 'text') {
```

Comment:
How about making this a switch case? If it will grow, it will end up in spaghetti code.